core-bundles/meta-model/org.eclipse.vorto.core/src/org/eclipse/vorto/core/api/model/BuilderUtils.java

```diff
@@ -126,9 +126,7 @@ public ModelBuilder(T model) {
     }
 
     public ModelBuilder<T> withVortolang(String lang) {
-      if ("1.0".equals(lang)) {
```

Comment:
Not sure, but since 1.0 is removed from the grammar validation, doesn't this check become more relevant now?

Reply:
It should not be there and there's another merge issue. pfffff
Thanks for noticing it.